tests/testthat/test-textstat_readability.R

```diff
@@ -145,4 +145,10 @@ test_that("textstat_readability works for renamed Bormuth.MC and Coleman.Liau.EC
                       textstat_readability(data_char_sampletext, measure = 'Bormuth.MC'))
     expect_identical(textstat_readability(data_char_sampletext, measure = 'Coleman.Liau'),
                       textstat_readability(data_char_sampletext, measure = 'Coleman.Liau.ECP'))
-})
\ No newline at end of file
+})
+
+test_that("textstat_readability raises error for non-included measures",{
```

Comment:
@kbenoit I have no idea why this test is not even though the expected and actual messages are identical. I've skipped it in this commit to ensure that the package builds sucessfully, but I am running into this error on my console:

```r
Error: Test failed: 'textstat_readability raises error for non-included measures'
* `quanteda::textstat_readability(data_char_sampletext, measure = "Gibberish")` threw an error
 with unexpected message.
Expected match: "Invalid measure(s): Gibberish"
Actual message: "Invalid measure(s): Gibberish"
```

Reply:
😅 It's because the "(" is a regular expression character.

Reply:
😅 thanks!